tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -3491,20 +3503,40 @@ <h2>Show similarity to selected datapoint</h2>
             return b.meanAbsError - a.meanAbsError;
           } else if (this.selectedFeatureSort == 'Mean squared error') {
             return b.meanSquaredError - a.meanSquaredError;
+          } else if (this.selectedFeatureSort == 'Median error') {
+            return b.medianError - a.medianError;
+          } else if (this.selectedFeatureSort == 'Median absolute error') {
+            return b.medianAbsError - a.medianAbsError;
+          } else if (this.selectedFeatureSort == 'Median squared error') {
+            return b.medianSquaredError - a.medianSquaredError;
           }
         });
       },
 
       fillInRegressionStats: function(regressionStats, name) {
         // From initial regression stats having regression error per example,
-        // calculate mean error, mean abs error, and mean squared error.
+        // calculate error statistics.
         function mean(data) {
           const sum = data.reduce((sum, value) => {
             return sum + value;
           }, 0);
 
           return sum / data.length;
         }
+        function median(sortedData) {
+          if (sortedData.length == 0) {
```

Comment:
mean does not have this check, is it necessary there as well?

Reply:
mean will return NaN already due to its use of reduce with an initial value of 0. Since median uses indexing instead of a reduce fn, it needs the check.